src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx

```diff
@@ -141,7 +141,7 @@ const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum, projLead, projMana
         <Button
           variant="danger"
           data-testId="deleteButton"
-          disabled={!hasPermissions}
+          disabled={!hasPermissions && !(risk.createdBy.userId === userId)}
```

Comment:
`risk.createdBy.userId !== userId` is probably better but I'm nitpicking

Reply:
lol true